tests/src/common/Wsk.scala

```diff
@@ -149,9 +149,14 @@ trait ListOrGetFromCollection extends FullyQualifiedNames {
      */
     def get(
         name: String,
+        summary: Boolean = false,
         expectedExitCode: Int = SUCCESS_EXIT)(
             implicit wp: WskProps): RunResult = {
-        cli(wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name)), expectedExitCode)
+        var cliparms = wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name))
+        if (summary) {
+            cliparms = cliparms ++ Seq("--summary")
+        }
+        cli(cliparms, expectedExitCode)
```

Comment:
@mdeuser here's the best of both worlds perhaps: no var, and allows for clearly stacking additional parameters:

```
    def get(
        name: String,
        summary: Boolean,
        expectedExitCode: Int = SUCCESS_EXIT)(
            implicit wp: WskProps): RunResult = {
        val params = Seq(noun, "get", "--auth", wp.authKey) ++
            { if (summary) Seq("--summary") else Seq() } ++
            Seq(fqn(name))
        cli(wp.overrides ++ params, expectedExitCode)
    }
```